client.go

```diff
@@ -58,12 +59,12 @@ func NewClient(host string, interval int) (cl *Client, err error) {
 	cl = &Client{
 		hostURL:    host,
 		http:       &http.Client{},
-		logger:     log.New(os.Stderr, "", log.LstdFlags), // Stderr logging
+		logger:     log.New(os.Stderr, "", log.Ldate|log.Ltime|log.Lshortfile),
 		interval:   intDur,
 		input:      make(chan MetricWithValue),
 		stop:       make(chan bool, 1),
 		stopped:    make(chan bool, 1),
-		metrics:    make(map[Metric]int, 1000),
+		metrics:    make(map[Metric]Value),
```

Comment:
This changed from a buffered channel to an unbuffered one. Intentional or accidental?